pypeit/core/datacube.py

```diff
@@ -542,27 +545,39 @@ def compute_weights(all_ra, all_dec, all_wave, all_sci, all_ivar, all_idx, white
     return all_wghts
 
 
-def coadd_cube(files, parset, overwrite=False):
+def coadd_cube(files, spectrograph=None, parset=None, overwrite=False):
     """ Main routine to coadd spec2D files into a 3D datacube
 
     Args:
-        files (list):
+        files (:obj:`list`):
             List of all spec2D files
-        parset (:class:`pypeit.par.core.PypeItPar`):
+        spectrograph (:obj:`str`, :class:`~pypeit.spectrographs.spectrograph.Spectrograph`, optional):
+            The name or instance of the spectrograph used to obtain the data.
+            If None, this is pulled from the file header.
+        parset (:class:`~pypeit.par.pypeitpar.PypeItPar`, optional):
```

Comment:
Explain the behaviour if parset is None in the docs, i.e. it uses the default parset based on the spectrograph.

Reply:
Done.